Azure/azure-cli
# {Find} az find: Change telemetry settings

**Description**  

We wanted to update the find module to use a more relevant piece of telemetry and respect the user set telemetry flag.

No user facing changed. If the flag is false, the service is sent empty strings as telemetry. 

**Testing Guide**  

No new commands or tests. Local testing was done to ensure this feature works.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
